src/main/pages/setup-kubernetes/kubernetes-multi-user.adoc

```diff
@@ -1,80 +1,87 @@
 
-The context of the commands below is `che/deploy/kubernetes/helm/che`
+The context of the commands below is `che/deploy/kubernetes/helm/che`.
 
 [id="to-deploy-with-dedicated-keycloak-as-authentication-server"]
-=== To deploy with dedicated keycloak as authentication server:
+=== Deploying with dedicated Keycloak as authentication server
+
+To deploy multi-user with dedicated Keycloak as authentication server:
 
-* Override default values by changing the values.yaml file and then typing:
+. Override default values by changing the `values.yaml` file. On the command line, run the following command:
 +
 ----
 helm upgrade --install <my-che-installation> --namespace <my-che-namespace> -f ./values/multi-user.yaml ./
 ----
-* Or, you can override default values during installation, using the `--set` flag:
++
+Or, override default values during installation, using the `--set` flag:
 +
 ----
 helm upgrade --install <my-che-installation> --namespace <my-che-namespace> -f ./values/multi-user.yaml --set global.ingressDomain=<my-hostname> --set cheImage=<my-image> ./
 ----
+
 * Master: `https://che-<che-namespace>.domain`
 * Keycloak: `https://keycloak-<che-namespace>.domain`
 * Workspaces servers: `https://server-host.domain`
 
 [id="to-deploy-without-keycloak-in-multi-user-mode"]
-=== To deploy without keycloak in multi-user mode
+=== Deploying without Keycloak in the multi-user mode
 
-In this case you have to supply custom OpenIdConnect provider (see https://github.com/eclipse/che-docs/blob/b2310017b1a75901cbec3b9c665d7ffa1cb23177/src/main/pages/setup-openshift/openshift-config.md[here] for more details), using the following flags:
+Supply the custom OpenIdConnect provider (see https://github.com/eclipse/che-docs/blob/b2310017b1a75901cbec3b9c665d7ffa1cb23177/src/main/pages/setup-openshift/openshift-config.md[here] for more details), using the following flags:
 
 ----
 helm upgrade --install <my-che-installation> --namespace <my-che-namespace> -f ./values/multi-user.yaml --set global.ingressDomain=<my-hostname>,cheImage=<my-image>,global.cheDedicatedKeycloak=false,customOidcProvider=<oidc-url>,cheKeycloakClientId=<oidc_clientId>,customOidcUsernameClaim=<user_name_claim> ./
 ----
 
-* cheKeycloakClientId - is your authentication server client ID
-* customOidcUsernameClaim - is optional, and should be used if your authentication server does not support 'preferred_username' claim in the JWT token (This can be checked by accessing oidc-url/.well-known/openid-configuration response). This parameter defines, which claim should be used to determine user name instead.
+Here:
+
+* `cheKeycloakClientId` is your authentication server client ID.
+* `customOidcUsernameClaim` is optional. Use if your authentication server does not support `preferred_username` claim in the JWT token (check this by accessing the oidc-url/.well-known/openid-configuration response). This parameter defines that the claim should be used to determine user name instead.
```

Comment:
Should "oidc-url/.well-known/openid-configuration" be styled?

Reply:
Yes, it should have been. I styled it. Thanks!